governance/tsc/tsc-2024-09-23.md

```diff
@@ -0,0 +1,217 @@
+
+nico: would it be possible for webrender to expose the glyph cache
+
+martin: there would be needed some rewrites
+
+martin: my idea is that we don't have to do all these things, and we use the thing that is already done; or then we switch to vello or tiny-skia when it's ready
+
+martin: it's different than SVG, because when we write the SVG code is to stay there forever
+
+nico: I'm concern about build requirements for skia
+
+nico: I'd not strongly oppose
+
+martin: I guess we would need to check if it's a problem in some platforms
+
+rego: Do have any plan for SVG? Do we want to do the weekend experiment?
+
+martin: probably giving 1-2 weeks to try the SVG support makes sense, for the SVGElement error, we might want to workaround it meanwhile
+
+nico: would it be possible to have the SVG code on its own crate?
+
+nico: The other thing I was wondering if we build our own SVG support. Would it be in its own crate? Will it be decoupled?
+
+martin: initially could be more decoupled but it'd get more and more into the layout code
+
+nico: I guess it can be possible to de-couple layout as well. In blitz we have text and box layout entirely decoupled. Maybe that could be the same for SVG?
+
+rego: Trying to decouple things sounds like a good long term goal.
+
+### License approval of two small non-MPL2 crates
+
+rego: https://github.com/servo/project/issues/101#issuecomment-2346616613
+
+rego: There was a comment on the agenda issue about integrating these two small crates with different licenses. Does anyone have a problem with that.
+
+nico: I had looked and it seems that this isn't request to relicense, but to move code from outside to inside. We've got lots of other crates that have a similar license.
+
+rego: I also think it's fine.
+
+jonathan: Someone should double-check that it is actually carrying the accurate license. I guess if it goes under the Servo organization, someone should double-check to make sure.
+
+martin: I find a bit odd the request, as the code in servo there are many with different licenses
+
+martin: I don't think we have any rules about the license of the code in the organziation, other than being open source
+
+martin: it seems pretty straight forward, I don't know the problem
+
+martin: even if it's not in Servo organization, it's going to be part of Servo anyway
+
+jonathan: Do we have an official policy? As a user of is the applied license MPL, since that's the stronger license.
+
+martin: I don't think there's an official policy, maybe when Servo was part of Mozilla, probably code from Mozilla had to be MPL
+
+martin: I don't think we've ever discussed a license policy
+
+rego: I don't think so, but we could have that discussion at another time.
+
+rego: We had other dependencies with similar licenses?
+
+nico: Do we know who the maintainers of rusturl are? On one hand, don't see problems with other repos, but on the other hand why doesn't it just go into the rust-url repository like other crates?
```

Comment:
cc @nicoburns I believe it _will_ go in the rust-url repo, but it's still a new crate publish so Henri wished to explicitly check the license